File: 우현/탐욕/2839_설탕배달.py
# 2839

kg = int(input())
count = 0
x=1

while(kg >=0):
    if kg%5 == 0:
        count = count + kg//5
        print(count)
        break
    kg = kg - 3
    count = count + 1
else:
    print(-1)
        

File: 지현/그리디/2839 설탕배달.swift
//
//  main.swift
//  1543 문서 검색
//
//  Created by 김지현 on 2022/05/09.
//

var num = Int(readLine()!)!
var count = 0

while true {
    if num % 5 == 0 {
        print(num/5 + count)
        break
    } else {
        num -= 3
        count += 1
        if num == 0 {
            print(count)
            break
        } else if num < 0 {
            print(-1)
            break
        }
    }
}
